great_expectations/rule_based_profiler/parameter_builder/value_set_multi_batch_parameter_builder.py

```diff
@@ -125,15 +126,25 @@ def _build_parameters(
                 FULLY_QUALIFIED_PARAMETER_NAME_ATTRIBUTED_VALUE_KEY
             ]
         )
+        details: dict = parameter_node[FULLY_QUALIFIED_PARAMETER_NAME_METADATA_KEY]
+
+        unique_values: Set[Any] = _get_unique_values_from_nested_collection_of_sets(
+            collection=metric_values
+        )
+        unique_values_as_array: np.ndarray = np.asarray(unique_values)
+        if unique_values_as_array.ndim == 0:
```

Comment:
Confirming my understanding - if `unique_values` is a scalar, `np.asarray` will only return a scalar. This condition will turn the array into an `np.ndarray` of dimension 1.